src/server/routes/actions.spec.ts

```diff
@@ -11,158 +12,115 @@ import request from "supertest";
 import { BotRequestType, FaucetRequestType } from "../../types";
 import router from "./actions";
 
-jest.mock("../services/ActionStorage");
-jest.mock("../services/Actions", () => {});
+jest.mock("../../dripper/DripperStorage");
+jest.mock("../../dripper/polkadot/PolkadotActions", () => {});
 jest.mock("../../logger", () => {
   return { logger: { error: mockLoggerError } };
 });
 
-jest.mock("../services/DripRequestHandler", () => {
+jest.mock("../../dripper/DripRequestHandler", () => {
   return {
+    getDripRequestHandlerInstance: jest.fn().mockImplementation(() => {
+      return { handleRequest: mockHandleRequest };
+    }),
     DripRequestHandler: jest.fn().mockImplementation(() => {
       return { handleRequest: mockHandleRequest };
     }),
   };
 });
-jest.mock("../config", () => {
-  return {
-    config: {
-      Get: mockConfig.mockImplementation((type: string) => {
-        switch (type) {
-          case "RPC_ENDPOINT":
-            return "http://localhost";
-          case "INJECTED_TYPES":
-            return "{}";
-          case "FAUCET_ACCOUNT_MNEMONIC":
-            // random seed phrase
-            return "scrub inquiry adapt lounge voice current manage chief build shoot drip liar head season inside";
-          default:
-            return "generic";
-        }
-      }),
-    },
-  };
+
+const mockConfigImplementation = (type: string) => {
+  switch (type) {
+    case "RPC_ENDPOINT":
+      return "http://localhost";
+    case "INJECTED_TYPES":
+      return "{}";
+    case "FAUCET_ACCOUNT_MNEMONIC":
+      // random seed phrase
+      return "scrub inquiry adapt lounge voice current manage chief build shoot drip liar head season inside";
+    case "EXTERNAL_ACCESS":
+      return true;
+    default:
+      return "generic";
+  }
+};
+
+jest.mock("../../config", () => {
+  return { serverConfig: { Get: mockConfig.mockImplementation((type: string) => mockConfigImplementation(type)) } };
 });
 
 let app: Express;
 
 const parameterError = (parameter: keyof BotRequestType | keyof FaucetRequestType) =>
   `Missing parameter: '${parameter}'`;
 
-describe("/drip/* tests", () => {
+describe("/drip/web tests", () => {
   beforeEach(() => {
-    jest.resetAllMocks();
```

Comment:
It is still working without this? 😮

Reply:
`resetAllMocks` is actually super confusing =)
`resetAllMocks` and `clearAllMocks` do opposite things from what they are named. 
`resetAllMocks` drops everything including implementation, while `clearAllMocks` only resets the state.

Reply:
Everyday I learn something new.

Thanks for the lesson!

Reply:
Yeah, that's a good lesson!
I have added `clearAllMocks` there - since I believe that was the original intention.